### PR TITLE
updated restrictions when adding a converter to the converter registry

### PR DESCRIPTION
**Pull request by yudilevi**

### CODE REVIEW

File: solidity/contracts/converter/BancorConverterRegistry.sol [hunk 1]
```diff
@@ -186,13 +186,12 @@ contract BancorConverterRegistry is IBancorConverterRegistry, ContractRegistryCl
...
     /**
       * @dev adds an existing converter to the registry
-      * anyone can add an existing converter to the registry, as long as the converter is valid
-      * note that a liquidity pool converter can be added only if no converter with the same reserve-configuration is already registered
+      * can only be called by the owner
       * 
       * @param _converter converter
     */
-    function addConverter(IBancorConverter _converter) public {
-        require(isConverterValid(_converter) && !isSimilarLiquidityPoolRegistered(_converter));
```
Comment (barakman) [inline solidity/contracts/converter/BancorConverterRegistry.sol:195]: @yudilevi : There's a reason why `isSimilarLiquidityPoolRegistered` has been implemented separately from `isConverterValid`, so the fact that you removed it is troublesome. Can you please explain?

File: solidity/contracts/converter/BancorConverterRegistry.sol [hunk 2]
```diff
@@ -388,16 +387,9 @@ contract BancorConverterRegistry is IBancorConverterRegistry, ContractRegistryCl
     function isConverterValid(IBancorConverter _converter) public view returns (bool) {
         // verify the the smart token has a supply and that the converter is active
         ISmartToken token = ISmartTokenController(_converter).token();
-        if (token.totalSupply() == 0 || token.owner() != address(_converter))
+        if (token.owner() != address(_converter))
```
Comment (barakman) [inline solidity/contracts/converter/BancorConverterRegistry.sol:390]: @yudilevi : Well, the entire function can be reduced to `return token.owner() == address(_converter);`.

File: solidity/contracts/converter/BancorConverterRegistry.sol (hunk 1 above)
Comment (yudilevi) [reply solidity/contracts/converter/BancorConverterRegistry.sol:195]: we discussed it - feel free to resolve

File: solidity/contracts/converter/BancorConverterRegistry.sol (hunk 2 above)
Comment (yudilevi) [reply solidity/contracts/converter/BancorConverterRegistry.sol:390]: Isn't it the case? Do you mean one liner to get the token and compare?

Comment (barakman) [reply solidity/contracts/converter/BancorConverterRegistry.sol:390]: @yudilevi : There's an `if`/`else` here. I say - let's go for a one-liner function.

**Pull request merged**